benchmark/run_all_benchmarks.sh

```diff
@@ -107,15 +116,18 @@ run_solver_benchmarks() {
 run_preconditioner_benchmarks() {
     [ "${DRY_RUN}" == "true" ] && return
     local bsize
-    for bsize in {1..32}; do
-        echo -e "\t\t running jacobi for block size ${bsize}/32" 1>&2
-        cp "$1" "$1.imd" # make sure we're not loosing the original input
-        ./preconditioner/preconditioner \
-            --backup="$1.bkp" --double_buffer="$1.bkp2" \
-            --executor="${EXECUTOR}" --preconditioners="jacobi" \
-            --max_block_size="${bsize}" \
-            <"$1.imd" 2>&1 >"$1"
-        keep_latest "$1" "$1.bkp" "$1.bkp2" "$1.imd"
+    for bsize in ${BLOCK_SIZES}; do
```

Comment:
This is interesting, does this work? With my bash while directly calling {1..32} here works, for the variable version to work I need to do something like:

```bash
for bsize in $(eval echo ${BLOCK_SIZES})
```

Reply:
I need to do an `eval` as well for it to work. Additionally adding quotes would be good.

Reply:
TBH, I didn't try the version with eval, just assumed it worked. I only used the preconditioner benchmarks, which replace this with only one value. I'll just define it as `$(seq 1 32)` so we're sure it works with any version of bash.

Reply:
Should work now